### ISSUE TITLE
Robot not displayed in Moveit Setup Assistant Noetic

### ISSUE THREAD

**Issue by j23saw** [ros-planning/moveit#2966]
### Description
Hello,
I tried to use moveit setup assistant with open manipulator for learning but after loading it didn't show up in the display so I tried it with the franka_description and have the same issue. I saw similar issue for ros-indigo and did try exporting export $LC_NUMERIC but it didn't change anything.

### Your environment
* ROS Distro: [Noetic]
* OS Version: Ubuntu 20.04 (WSL2)
* Source or Binary build? Binary
* If binary, which release version? 1.1.5
* If source, which branch?

### Steps to reproduce
I cloned the franka_description while following the setup-assistant tutorial on https://ros-planning.github.io/ and loaded the panda_arm_.urdf.xacro file. The setup happens and all the collisions are shown in the second tab but it is not displayed in the visual panel.

### Backtrace or Console output
[backtrace](https://gist.github.com/j23saw/1cf19ef05b9ba5e98ef647e57fb750cd)

**Comment by rhaschke** [comment]
Can you see a robot / anything in rviz, in general? Maybe, the problem is related to your WSL2 setup.

**Comment by j23saw** [comment]
Oh no I can't see it in rviz too, you may be right it might be related to my WSL2 setup. Any pointers to debug this problem?

**Comment by simonschmeisser** [comment]
First please check in rviz if you can see any of the generic stuff and also if the `RobotDisplay` works

**Comment by j23saw** [comment]
I can't see any RobotDisplay option here in rviz

**Comment by rhaschke** [comment]
`RobotModel` display?

**Comment by j23saw** [comment]
There's no RobotModel display but I can see "Scene Robot"

**Comment by rhaschke** [comment]
![image](https://user-images.githubusercontent.com/5376030/142775867-8f46a08a-3c21-449d-a79f-4f1244770d27.png)

**Comment by j23saw** [comment]
Ah yes, I added RobotModel but still it can't display the actual 3d model, only collision is visible

**Comment by rhaschke** [comment]
Maybe you are running into this issue: https://answers.ros.org/question/272692/rviz-dows-not-display-model-urdf-tutorial?
I thought this was fixed...

**Comment by rhaschke** [comment]
Collision and visual models of franka_description use different approaches: visual models are .dae files, collision models are basic geometry primitives, directly rendered by rviz. Maybe you have an issue with the `assimp` package as well? This is used to load meshes.

**Comment by simonschmeisser** [comment]
> Maybe you are running into this issue: https://answers.ros.org/question/272692/rviz-dows-not-display-model-urdf-tutorial? I thought this was fixed...

Yes, I fixed that some years ago and they actually also tried setting LC_NUMERIC so that should not be the issue here

**Comment by simonschmeisser** [comment]
@lilustga do you have any experience with WSL2?

Are there no error messages at call on the command line?

**Comment by lilustga** [comment]
@simonschmeisser I do have experience with WSL2 and ROS. Have not seen this problem before though

**Comment by mashuai191** [comment]
looks quite similar with my reported  #3006 , and my env is also WSL2, on win11
